GoogleDataTransport/GDTCORLibrary/GDTCORPlatform.m

```diff
@@ -71,7 +71,16 @@ GDTCORNetworkMobileSubtype GDTCORNetworkMobileSubTypeMessage() {
     };
     networkInfo = [[CTTelephonyNetworkInfo alloc] init];
   });
-  NSString *networkCurrentRadioAccessTechnology = networkInfo.currentRadioAccessTechnology;
+  NSString *networkCurrentRadioAccessTechnology;
+  if (@available(iOS 12, *)) {
+    NSDictionary<NSString *, NSString *> *networkCurrentRadioAccessTechnologyDict =
+        networkInfo.serviceCurrentRadioAccessTechnology;
+    if (networkCurrentRadioAccessTechnologyDict) {
+      networkCurrentRadioAccessTechnology = networkCurrentRadioAccessTechnologyDict.allValues[0];
```

Comment:
Based on the [documentation for this method](https://developer.apple.com/documentation/coretelephony/cttelephonynetworkinfo/3024510-servicecurrentradioaccesstechnol?language=objc) we should be checking the keys and not the values. 

>The keys for the `serviceCurrentRadioAccessTechnology` dictionary are `NSString` objects, each of which represents a service. The entry associated with a key is nil if the service is not registered on any network.
>
>Although the actual value of a key isn’t important, you can also use it to get the carrier information associated with the service. To do so, pass the key to the `serviceSubscriberCellularProviders` dictionary.

Reply:
Thank you, Morgan. I have changed the if condition to check dictionary.count. And for the networkCurrentRadioAccessTechnology string we want is stored in the values.